pallets/subtensor/src/macros/hooks.rs

```diff
@@ -115,7 +115,9 @@ mod hooks {
                 // Reset max burn
                 .saturating_add(migrations::migrate_reset_max_burn::migrate_reset_max_burn::<T>())
                 // Migrate ColdkeySwapScheduled structure to new format
-                .saturating_add(migrations::migrate_coldkey_swap_scheduled::migrate_coldkey_swap_scheduled::<T>());
+                .saturating_add(migrations::migrate_coldkey_swap_scheduled::migrate_coldkey_swap_scheduled::<T>())
+                // Clear epoch storage values for root netuid
+                .saturating_add(migrations::migrate_clear_root_epoch_values::migrate_clear_root_epoch_values::<T>());
```

Comment:
I think we should wait for #1660 to be merged because the weights are not accounted for atm